tests/test_logs.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+"""
+Test custom logger
+"""
+
+
+class Test_Logs(unittest.TestCase):
+    """Import modmail"""
+
+    def test_import(self):
+        """
+        Import modmail
+        """
+        from modmail import log
+
+        self.assertTrue(True)
```

Comment:
Can we use importlib to import the module functionally? Might look nicer.

Reply:
How would this work?